src/portfolio.js

```diff
@@ -9,22 +9,24 @@ const greeting = {
   /* Your Summary And Greeting Section */
   username: "Saad Pasta",
   title: "Hi all, I'm Saad",
-  subTitle: emoji("A passionate Software Developer 🚀 having an experience of building Web and Mobile applications with JavaScript / Reactjs / Nodejs / React Native and some other cool libraries and frameworks."),
-  resumeLink: "https://drive.google.com/file/d/1ofFdKF_mqscH8WvXkSObnVvC9kK7Ldlu/view?usp=sharing"
+  subTitle: emoji("A passionate Full Stack Software Developer 🚀 having an experience of building Web and Mobile applications with JavaScript / Reactjs / Nodejs / React Native and some other cool libraries and frameworks."),
+  resumeLink: "https://drive.google.com/file/d/1ofFdKF_mqscH8WvXkSObnVvC9kK7Ldlu/view?usp=sharing",
+  displayHeader: true, // Set true to display header section, defaults to false
```

Comment:
@Exspiravit Look's like you didn't get my point. 
Remove Name of the Section from the header if the section is displayed false.
For E.g 
Is Education Section is `display: false` **Education**  key should not appear in the header.
Please remove this key. 

Sorry for not clearing myself before